message InnerType {
    optional uint32 in_a = 1;
    optional uint32 in_b = 2;

    enum EnumType {
        Type0 = 0;
        Type1 = 1;
        Type2 = 2;
    }

    optional EnumType et1 = 3;
    optional EnumType et2 = 4;
}

message NewStyle {
    optional uint32 planid = 1;
    optional uint64 unitid = 2;
    repeated uint32 mtid = 3;
    optional InnerType it = 4;
    optional string url = 5;
}
